netdata-installer.sh

```diff
@@ -814,8 +814,18 @@ install_go() {
 		done
 		tmp=$(mktemp -d /tmp/netdata-go-XXXXXX)
 		GO_PACKAGE_BASENAME="go.d.plugin-$GO_PACKAGE_VERSION.$OS-$ARCH"
+
 		download "https://github.com/netdata/go.d.plugin/releases/download/$GO_PACKAGE_VERSION/$GO_PACKAGE_BASENAME" "${tmp}/$GO_PACKAGE_BASENAME"
+		if [ $? ] ; then
```

Comment:
This is plain wrong. Checking of exit code of previous command with `if` statement is discouraged in BASH, look here: https://github.com/koalaman/shellcheck/wiki/SC2181

Also if anything, this should be written as `if [ $? -ne 0 ] ; then`

Reply:
Additionally this whole logic shouldn't exist as this should be handled by `run()` function which is used by `download()` function.